tools/fregrid/globals.h

```diff
@@ -236,4 +236,15 @@ typedef struct{
   double *f_min;
 } Monotone_config;
 
+typedef struct{
+  double *lat_min_list;
+  double *lat_max_list;
+  double *lon_min_list;
+  double *lon_max_list;
+  double *lon_avg;
+  double *lon_list;
+  double *lat_list;
+  int *n_list;
+} Minmaxavglists;
```

Comment:
If you use a refactoring tool, it may be better to name this as
``Minmax_avg_lists``  ?